consensus/expected_test.go

```diff
@@ -222,12 +233,6 @@ func TestIsWinningTicket(t *testing.T) {
 		wins       bool
 	}{0x00, 1, 5, true}
 
-	minerAddress := address.NewForTestGetter()()
-	ctx := context.Background()
-	d := datastore.NewMapDatastore()
-	bs := blockstore.NewBlockstore(d)
-	var st state.Tree
-
 	t.Run("IsWinningTicket returns false + error when we fail to get total power", func(t *testing.T) {
 		ptv1 := NewFailingTestPowerTableView(testCase.myPower, testCase.totalPower)
 		ticket := [sha256.Size]byte{}
```

Comment:
I believe that this is a different size (32) than the signature size mentioned above (64). If this is correct is it weird that IsWinningTicket doesn't throw an error?

Reply:
It doesn't, because it just copies the slice bytes into the Signature; however, it's kinda sloppy to make it a `[sha256.Size]byte{}`, so I'll at least declare the ticket as a types.Signature.